src/execute/oas3/style-serializer.js

```diff
@@ -1,35 +1,19 @@
 const isRfc3986Reserved = (char) => ":/?#[]@!$&'()*+,;=".indexOf(char) > -1;
-const isRrc3986Unreserved = (char) => /^[a-z0-9\-._~]+$/i.test(char);
+const isRfc3986Unreserved = (char) => /^[a-z0-9\-._~]+$/i.test(char);
 
 // eslint-disable-next-line default-param-last
-export function encodeDisallowedCharacters(str, { escape } = {}, parse) {
-  if (typeof str === 'number') {
-    str = str.toString();
-  }
-
-  if (typeof str !== 'string' || !str.length) {
-    return str;
-  }
-
-  if (!escape) {
-    return str;
-  }
-
-  if (parse) {
-    return JSON.parse(str);
-  }
-
+export function encodeCharacters(str, { characterSet = 'reserved' } = {}) {
```

Comment:
I'm wondering if we could have the signature for this as just 
```
encodeCharacters(str, characterSet = 'reserved')
```

Reply:
We would. I guess the original author of the function expected that the second argument the be an `Options` objects and that it would eventually support more options. But as there is only one option currently  and given the function is internal, you can (if you wish) transform the signature to standard default parameter.